src/main/java/com/ita/if103java/ims/service/impl/EventServiceImpl.java

```diff
@@ -0,0 +1,47 @@
+package com.ita.if103java.ims.service.impl;
+
+import com.ita.if103java.ims.dao.EventDao;
+import com.ita.if103java.ims.dto.EventDto;
+import com.ita.if103java.ims.entity.Event;
+import com.ita.if103java.ims.mapper.EventDtoMapper;
+import com.ita.if103java.ims.service.EventService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.stereotype.Service;
+
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+import java.util.List;
+import java.util.Map;
+
+@Service
+@PropertySource("classpath:application.properties")
+public class EventServiceImpl implements EventService {
+
+    private EventDao eventDao;
+    private EventDtoMapper eventDtoMapper;
+
+    @Autowired
+    public EventServiceImpl(EventDao eventDao, EventDtoMapper eventDtoMapper) {
+        this.eventDao = eventDao;
+        this.eventDtoMapper = eventDtoMapper;
+    }
+
+    @Override
+    public EventDto create(EventDto eventDto) {
```

Comment:
This method will be called internally from other classes, so it may receive Entity as argument.
Please mark this method @Async and configure Spring to work with asynchronous methods.

Also this method should return void, so line 34 is not really needed